pkg/config/env/mongo.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // MongoConnectionString returns Athens Mongo Storage connection string defined by ATHENS_MONGO_CONNECTION_STRING
 func MongoConnectionString() (string, error) {
-	env, err := envy.MustGet("ATHENS_MONGO_CONNECTION_STRING")
+	env, err := envy.MustGet("ATHENS_MONGO_CONNECTION_STRING", "mongodb://127.0.0.1:27017")
 	if err != nil {
```

Comment:
Then this error check will no longer be needed. And the function signature updated.